src/generators/man-page/template.1

```diff
@@ -1,10 +1,9 @@
 .\"
-.\" This file is automatically generated by api-docs-tooling.
-.\" Do not edit this file directly. Please make changes to CLI.md
-.\" and then regenerate this file.
+.\" This file was generated automatically by the api-docs-tooling tool.
+.\" Please do not edit this file manually. Make any updates to cli.md
+.\" and regenerate the file afterward.
 .\"
-.\" For generation instructions using api-docs-tooling, see:
-.\" https://github.com/nodejs/api-docs-tooling
+.\" To regenerate this file, run `make doc/node.1`.
```

Comment:
**Review note**: The `make doc/node.1` command is being added in https://github.com/nodejs/node/pull/55268